packages/mobile/package.json

```diff
@@ -25,30 +25,28 @@
     "@quiet/backend": "^0.16.0",
     "@quiet/identity": "^0.15.0",
     "@quiet/state-manager": "^0.16.0",
-    "@react-native-async-storage/async-storage": "^1.15.4",
-    "@react-native-community/art": "^1.2.0",
     "@react-navigation/native": "^6.0.0",
     "@react-navigation/native-stack": "^6.9.0",
-    "@reduxjs/toolkit": "^1.5.1",
+    "@reduxjs/toolkit": "^1.9.1",
     "buffer": "^6.0.3",
     "events": "^1.1.1",
     "i18next": "^20.2.2",
+    "jdenticon": "^3.2.0",
     "luxon": "^1.27.0",
     "pkijs": "*",
     "pvutils": "^1.0.17",
     "react": "18.1.0",
+    "react-dom": "18.1.0",
     "react-i18next": "^11.8.15",
     "react-linkify": "^1.0.0-alpha",
     "react-native": "0.70.4",
     "react-native-blob-util": "^0.16.2",
     "react-native-config": "^1.4.2",
     "react-native-crypto": "^2.2.0",
     "react-native-device-info": "^8.1.3",
-    "react-native-events": "^1.0.17",
     "react-native-fast-image": "^8.6.1",
     "react-native-fs": "^2.18.0",
     "react-native-gesture-handler": "^2.8.0",
-    "react-native-jdenticon": "0.0.3",
```

Comment:
this required older react-native-svg than we have in the project. I moved to code to component in our codebase